player/public/coreplugins/org.ekstep.overlay-1.0/renderer/controller/overlay.js

```diff
@@ -17,8 +17,11 @@ app.controllerProvider.register("OverlayController", function($scope, $rootScope
     $scope.showTeacherIns = true;
     $scope.showReload = true;
     $scope.showContentClose = false;
-    $scope.init = function() {
-
+    $rootScope.enableDefaultNav = true;
+    var pluginInstance = EkstepRendererAPI.getPluginObjs("org.ekstep.overlay");
+    $scope.defaultPreviousIcon =  pluginInstance.navPreviousIcon;
```

Comment:
This has to be handled in the template.html files. Template has to handle it's images & styles